packages/parcel-config/README.md

```diff
@@ -0,0 +1,13 @@
+# @compiled/parcel-config
+
+Compiled configuration for Parcel.
+
+## Installation
+
+```bash
+npm i @compiled/parcel-optimizer
+```
+
+## Usage
+
+Detailed docs and example usage can be [found on the documentation website](https://compiledcssinjs.com/docs/pkg-parcel).
```

Comment:
The page is currently blank. Are you also updating the docs?

Reply:
Yep I plan to! In retrospect, I should probably update this to https://compiledcssinjs.com/docs/pkg-parcel-config so it corresponds to a real package